apps/autostart-stop.html.md.erb

```diff
@@ -0,0 +1,83 @@
+
+```toml
+auto_stop_machines = true
+auto_start_machines = true
+```
+
+Existing apps are automatically started but not automatically stopped by default:
+
+```toml
+auto_stop_machines = false
+auto_start_machines = true
+```
+
+In most cases, we recommend setting `auto_stop_machines` and `auto_start_machines` to the same value to avoid having instances that either never start or never stop. 
+
+If `auto_start_machines = true` and `auto_stop_machines = false`, the proxy will start your instances but they will never be stopped automatically. If you have a reason to only stop instances manually, then these settings are just fine.
+
+If `auto_start_machines = false` and `auto_stop_machines = true`, the proxy will scale your instances down but won't be able to start them again. If all of your instances are scaled down, then requests will start failing.
+
+## How It Works
+
+The Fly Proxy runs the process to stop and start machines automatically every few minutes.
+
+When `auto_stop_machines = true` in your `fly.toml`, the proxy looks at machines running in a single region and uses the [`soft_limit` setting](/docs/reference/configuration/#the-http_service-section) for each machine to determine if there's excess capacity. If the proxy decides there's excess capcity, it will stop exactly one machine. The proxy repeats this process every few minutes, stopping only one machine per region, if needed, each time.
+
+Fly proxy considers all of the running instances of your app in a given region, such as `fra`, and determines excess capacity as follows:
+
+* If there's more than one instance in the region:
+  * the proxy determines how many instances are over their `soft_limit` setting and then calculates excess capacity: `excess capacity = num of instances - (num instances over soft limit + 1)`
+  * if excess capacity is 1 or greater, then the proxy stops 1 instance
+* If there's only one instance in the region:
+  * the proxy checks if the instance has any traffic
+  * if the instance has no traffic (a load of 0), then the proxy stops the instance
+
+When `auto_start_machines = true` in your `fly.toml`, the Fly Proxy restarts an instance in the nearest region when required.
+
+Fly proxy determines when to start an instance as follows:
+
+* The proxy waits for a request to your app.
+* If all running instances are above their `soft_limit` setting, then the proxy starts a stopped instance from the nearest region (if there are any stopped instances).
+* The proxy routes the request to the newly started instance.
+
+## When to Stop and Start Machines Automatically, or Not
+
+If your app has highly variable workloads, then you can set `auto_stop_machines` and `auto_start_machines` to true to manage your instances as demand decreases and increases. This could reduce costs, because you'll never have to run excess instances to handle peak load; you'll only run what's necessary at any given time.
```

Comment:
It's all about connections or requests to a configured service here (no CPU or RAM load taken into account here) -- so we should make that clear -- it's not a general workload.

true -> `true`; false-> `false` makes it easier to parse